### DISCUSSION TITLE
Page Layout with only Header or with only Footer Site Parts

### DISCUSSION THREAD

**Discussion by atcdomsol** [elementor/elementor#20044] [Feature Request]
I have a request that I think could be handy. I typically build landing pages on websites where I may want to use the global header but not the footer (and vice-versa).  

I usually use the Canvas template so I'm starting with a blank slate, but I may want to keep the global header or footer for the page. Using Canvas eliminates both. 

My request: 2 additional Template options that are available for choosing. 

Canvas - Header (displays header + blank slate)
Canvas - Footer (displays footer + blank slate)

I think this would be a shortcut/less work method instead of creating individual template areas or setting up a lot of "Display Options".

Thanks!

**Comment by nicholaszein** [comment]
Alternatively, you can use the Template widget (Pro) to display your Header or Footer templates anywhere. Keep them synchronized with the rest of the site without needing to have both or set Display Conditions!

Check it out!

https://github.com/elementor/elementor/assets/113993460/45a7e79a-ec06-4670-a725-b349b3a48114

**Reply by atcdomsol** [reply]
Thanks! I have been using global templates since they made it part of Elementor's core.
